net-irc/emech/emech-3.0.99_p3-r2.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit readme.gentoo-r1 toolchain-funcs
+
+DESCRIPTION="UNIX compatible IRC bot programmed in the C language"
+HOMEPAGE="http://www.energymech.net/"
+SRC_URI="http://www.energymech.net/files/${P/_/}.tar.gz"
+S="${WORKDIR}/${P/_/}"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug session tcl"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="virtual/libcrypt:="
+DEPEND="${RDEPEND}"
+
+DOC_CONTENTS="You can find a compressed sample config file at /usr/share/doc/${PF}"
+
+src_prepare() {
+	eapply "${FILESDIR}/${P}-buildfix.patch"
+	eapply "${FILESDIR}/${P}-fix-build-for-clang16.patch"
+
+	sed -i \
+		-e 's: "help/":"/usr/share/energymech/help/":' \
+		-e 's: "messages/":"/usr/share/energymech/messages/":' \
+		src/config.h.in || die
+	# Respect CFLAGS and LDFLAGS
+	sed -i \
+		-e '/^LFLAGS/s/\$(PIPEFLAG)/\0 \$(OPTIMIZE) \$(LDFLAGS)/' \
+		-e '/^GDBFLAG/d' \
+		-e '/^PIPEFLAG/d' \
+		src/Makefile.in || die
+
+	eapply_user
+}
+
+myconf() {
+	tc-export CC
+	echo ./configure $*
```

Comment:
Replace these two lines with edo, and fold myconf() into src_configure (just use an array)